LoopKitUI/Extensions/View+KeyboardAware.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  View+KeyboardAware.swift
+//  LoopKitUI
+//
+//  Created by Michael Pangburn on 7/22/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+
+// NOTE: In iOS 14, keyboard management is handled automatically in SwiftUI.
+extension View {
+    public func onKeyboardStateChange(perform updateForKeyboardState: @escaping (_ keyboardHeight: Keyboard.State) -> Void) -> some View {
+        onReceive(Keyboard.shared.$state, perform: updateForKeyboardState)
+    }
+
+    public func keyboardAware() -> some View {
+        modifier(KeyboardAware())
+    }
+}
+
+public struct KeyboardAware: ViewModifier {
```

Comment:
@novalegra I saw you had something similar in TidepoolService; I needed a bit more flexibility in separating the keyboard publisher from the ViewModifier to make other UI updates on the bolus screen, so I wrote the version here.

Reply:
This looks great - I'll probably update my screen to use this once your changes land since it's more flexible